### ISSUE TITLE
hellow

### ISSUE THREAD

**Issue by sizhenfei** [russellgeum/Phase-aware-Deep-Complex-UNet#2]
I use your program, but the training loss is a negative. can you help me? What should I do?

**Comment by y891530061** [comment]
i use loss function is wSDR ; neural network is dcunet20 ; input data is original speech signal,

**Comment by Doyosae** [comment]
wSDR loss is negative... is normal maybe.
If you refer to the formula of the paper, it is negative.
Am I mistaken?

**Comment by Doyosae** [comment]
> I use your program, but the training loss is a negative. can you help me? What should I do?
wSDR loss is negative... is normal maybe.
If you refer to the formula of the paper, it is negative.
Am I mistaken?

**Comment by y891530061** [comment]
> > I use your program, but the training loss is a negative. can you help me? What should I do?
> > wSDR loss is negative... is normal maybe.
> > If you refer to the formula of the paper, it is negative.
> > Am I mistaken?

I've trained 500 times with your program, I'm using the model that it saved last time. then . i used model_test.py to Enhance noisy speech. but, It sounds awful, So ,I think I might have done something wrong, How do your enhancement results？Here's a link to my results( https://drive.google.com/drive/folders/17QlbDEwRq3t0I8_xg20MnBw47hgrnJtx?usp=sharing )

**Comment by cheriylan** [comment]
> I use your program, but the training loss is a negative. can you help me? What should I do?

hello, I run model_train.py  but met this error  "cannot reshape array of size 161338 into shape (16384,1)",How did you solve it.

**Comment by y891530061** [comment]
> > I use your program, but the training loss is a negative. can you help me? What should I do?
> 
> hello, I run model_train.py but met this error "cannot reshape array of size 161338 into shape (16384,1)",How did you solve it.
这个程序跑出来也没结果，程序可能是错的，你还是换其他方法吧

**Comment by y891530061** [comment]
> > I use your program, but the training loss is a negative. can you help me? What should I do?
> 
> hello, I run model_train.py but met this error "cannot reshape array of size 161338 into shape (16384,1)",How did you solve it.

你看我那个问题，作者已经不回我了，增强的结果比不增强之前还低

**Comment by Joly2001** [comment]
> > I use your program, but the training loss is a negative. can you help me? What should I do?
> 
> hello, I run model_train.py but met this error "cannot reshape array of size 161338 into shape (16384,1)",How did you solve it.

您好！我也想问这个问题，你后来解决了吗？我是8000采样率
